src/main/target/EXF722DUAL/target.mk

```diff
@@ -0,0 +1,11 @@
+F7X2RE_TARGETS += $(TARGET)
+FEATURES    = VCP ONBOARDFLASH
+
+TARGET_SRC = \
+            drivers/accgyro/accgyro_fake.c \
```

Comment:
Not sure what the purpose of including the `_fake` drivers here is - they should not be used in production targets, as they obviously do not return useful data.

Reply:
Resolve it and rebase the pr.